assembl/static2/css/themes/default/_theme.scss

```diff
@@ -0,0 +1,46 @@
+/* 
+  ASSEMBL DEFAULT THEME
+  If you want to create a new theme, it is recommended that your theme SCSS file uses `@import` of this default theme file (for example, `@import "../../../default/_theme.scss";`), so if a default theme variable is not overriden in your theme (for example a variable which would have been added in Assembl code after the creation of your theme), the SCSS will still compile, and the variable will have the default value.
+*/
+
+@import "../../variables.scss";
+
+/*
+  Local theme variables
+  Ranked from light to dark
+
+  You can define here any variable that you want. This can be useful if you want several theme variables to have the same value.
+  Example:
+  In this Local theme variables section, you would add:
+  $th_local_brand_logo_red: $FF0000;
+  And in the Global theme variables section, you would redefine:
+  $first-color: $th_local_brand_logo_red;
+  $title1-dark-color: $th_local_brand_logo_red;
+
+  Variables in this section should only be used within this current theme file. NEVER refer to these color variables in other SCSS files.
+*/
+
+$th_local_brand_logo_red: #FF0000;
+
+
+/*
+  Global theme variables
+  You can redefine here any of the default theme variables. Theme variables are used in the entire SCSS infrastructure. Theme variables which are not redefined by a theme will keep their default value, which is defined in css/themes/default/_theme.scss and in css/variables.scss.
+*/
+
+$first-color: $th_local_brand_logo_red;
+$first-color-light: $th_local_brand_logo_red;
+$second-color: $th_local_brand_logo_red;
+$third-color: $th_local_brand_logo_red;
+$title1-dark-color: $first-color;
+$title2-dark-color: $first-color;
+$title3-dark-color: $first-color;
+$title4-dark-color: $first-color;
+$title5-dark-color: $grey4;
+$title1-light-color: $white;
+$title2-light-color: $white;
+$title3-light-color: $white;
+$title4-light-color: $white;
+$title5-light-color: $white;
```

Comment:
Yes I should probably document more about the processes related to themes handling (how to install the new repo when upgrading a server, how to add new theme variables, etc). But the "how to create a new theme" process remains the same as in v1. Other processes are also almost identical.